src/sentry/sentry_apps/external_requests/select_requester.py

```diff
@@ -50,16 +50,24 @@ def run(self) -> dict[str, Any]:
 
             response = json.loads(body)
         except Exception as e:
-            logger.info(
-                "select-requester.error",
-                extra={
-                    "sentry_app_slug": self.sentry_app.slug,
-                    "install_uuid": self.install.uuid,
-                    "project_slug": self.project_slug,
-                    "error_message": str(e),
-                    "url": url,
-                },
-            )
+            extra = {
+                "sentry_app_slug": self.sentry_app.slug,
+                "install_uuid": self.install.uuid,
+                "project_slug": self.project_slug,
+                "error_message": str(e),
+                "url": url or "",
+            }
+
+            if not url:
+                extra.update(
+                    {
+                        "uri": self.uri,
+                        "dependent_data": self.dependent_data,
+                        "webhook_url": self.sentry_app.webhook_url,
+                    }
+                )
```

Comment:
Alternatively could be just capturing the `uri`, `dependent_data` and `webhook_url`, and that would be more compact, but more stuff to log. :/ Don't really have an opinion